config/hubs/schema.yaml

```diff
@@ -198,8 +206,13 @@ properties:
                       "<clustername>-<hubname>".
 
                       See https://manage.auth0.com/dashboard/us/2i2c/connections/database.
-            required:
-              - connection
+            if:
```

Comment:
this is super cool, TIL! Thanks @damianavila!

Reply:
Yeah... I really liked it when I discovered it 😉